test/xpu_api/numerics/complex.number/cmplx.over/norm.pass.cpp

```diff
@@ -46,9 +46,9 @@ ONEDPL_TEST_NUM_MAIN
     test<float>();
     IF_DOUBLE_SUPPORT(test<double>())
     IF_LONG_DOUBLE_SUPPORT(test<long double>())
-    test<int>();
-    test<unsigned>();
-    test<long long>();
+    IF_DOUBLE_SUPPORT(test<int>();
```

Comment:
In all these cases type ```double``` used inside test.

Reply:
Let's discuss it in https://github.com/oneapi-src/oneDPL/pull/736/files#r1059356935